notebooks/Medical Image Generation with Generative AI.ipynb

```diff
@@ -88,7 +89,8 @@
     "cs_bucket = \"<bucket/container name to push the experiment job data to>\"  # Please fill it with the actual bucket name\n",
     "\n",
     "# Job configuration\n",
-    "num_gpu = 1                                                               # Number of GPUs to use"
+    "num_gpu = 1                                                               # Number of GPUs to use\n",
```

Comment:
Bundle training only supports 1 gpu

Reply:
Please remove this until we have the support for bundle training.

Reply:
It is not training @mingxin-zheng. It is only batch inference. Does that support multi gpu?

Reply:
Since ConfigWorkflow doesn't support mGPU in general, and we are using it for both training and inference, I think the support isn't there yet.

Reply:
Got it